SokolTextGame/Commands.cs

```diff
@@ -35,4 +64,14 @@ public void Execute(string[] words, World world)
             Console.WriteLine("I'm sorry, but there is no such command. =) ");
         }
     }
-}
+
+    public class CreatePlayer : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words[0].Equals("warrior")) world.player = new Warrior(new BareFists());
+            else if (words[0].Equals("rogue")) world.player = new Rogue(new BareFists());
+            else if (words[0].Equals("wizard")) world.player = new Wizard(new BareFists());            
```

Comment:
Also, please add `else throw new Exception("CreatePlayer.Execute(): unknown player type?!");` as the final condition if all else fails. Technically, this is unreachable code, but if in the future you somehow get a bug where the `World.player` member is null, it will be a 100 times easier to track it down with this exception in place.